katello-agent/katello-agent.spec

```diff
@@ -1,6 +1,6 @@
 Name: katello-agent
-Version: 2.5.0
-Release: 2%{?dist}
+Version: 2.5.1
+Release: 0%{?dist}
```

Comment:
ahhh, the build failed because we only bump the version when a new 'release' happens in the katello-agent repo.  Since this is purely a spec file change, simply bump the release and leave the version at '2.5.0'